pinot-servicemanager/Dockerfile

```diff
@@ -32,6 +32,9 @@ WORKDIR /opt/pinot
 RUN adduser -g '' -h ${PWD} -D ${USER}
 USER ${USER}
 
+# Copy generic avro serde
+COPY --chown=${USER} build/classes/java/main/ ./classes/
```

Comment:
See https://github.com/hypertrace/pinot/pull/33 for another pattern of providing plugins. Cleaner IMO, as this makes more assumptions - for example, how do any dependencies of these classes get provided.

Reply:
will follow the approach of #33.
we are making the usage of schema-registry optional in local mode deployment. In that context, we have defined a new serde which passes writer schema as part of the payload (https://github.com/hypertrace/kafka-streams-framework/pull/17/), the same will be used in pinot to deserialize data. In normal cases, we will still continue to use schema registry-based serde. So, we are keeping these changes for pinot-servicemanager.